.github/workflows/pkg-host-bun.yml

```
@@ -0,0 +1,23 @@
+
+jobs:
+  bun-host:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: oven-sh/setup-bun@v1
+        with:
+          bun-version: latest
+      - name: Bun host smoke
+        run: bun run packages/wesley-host-bun/examples/smoke.mjs | tee bun-host.out && grep -q '^BUN_HOST_OK:' bun-host.out
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)